src/App/Header/Logo.js

```diff
@@ -1,11 +1,12 @@
 import { Link } from 'react-router-dom';
 import logo from './logo.png';
+import { routes } from 'constants/routes';
```

Comment:
Absolute import should be before a relative one

Reply:
Done.